src/runner.ts

```diff
@@ -84,9 +84,10 @@ export function preRun(
     path: string,
     timeout: number
 ): Option<Execution> {
-    debug("pre-run", code);
+    debug("pre-run", code, output, path, timeout); //add more debug
 
-    if (checkMD5(code, path)) {
+    // check if md5 is same and wcmp is already compiled
+    if (checkMD5(code, path) && existsSync(output)) {
```

Comment:
Why do you need to check if output exists? In theory if it passes `checkMD5` file should exist, unless it was externally removed. I'm ok adding this here, since we should not depend on file being removed, but I'm curious if you already had some issue with this.